resources/css/_code.css

```diff
@@ -15,6 +15,15 @@ pre code.torchlight .summary-caret {
     @apply mr-1;
 }
 
+li code,
+p code {
+    @apply inline-flex max-w-full overflow-scroll
+}
+
+table {
+    @apply block max-w-full overflow-scroll
+}
+
```

Comment:
This is great! I think it should be moved to `app.css` as this file is specific to code snippets.

Reply:
That's cool! I've moved it to `app.css` but I did change `overflow-scroll` to `overflow-auto` I'll be leaving a comment comparing the two, with the reason why I did. I'm always open for feedback!